shepherd/RecipeManager.py

```diff
@@ -0,0 +1,77 @@
+from Utils import *
+
+
+class RecipeManager():
+
+    master_recipes = [RECIPE_PROBABILITIES.EASY_RECIPE, RECIPE_PROBABILITIES.MEDIUM_RECIPE, \
```

Comment:
I don't understand why this is a list of references to an empty list... it does give me a good idea of how to manage the recipes tho.

Reply:
The idea is to make it easier for us to load and reload the list by having a list that we can read from without the programmer looking back and forth
It's to make our lives and future people lives easier

Also, this is not empty (or I believe it shouldn't be)